src/Fission/IPFS/CID/Types.hs

```diff
@@ -46,6 +49,9 @@ instance MimeRender PlainText CID where
 instance MimeRender OctetStream CID where
   mimeRender _ = UTF8.textToLazyBS . unaddress
 
+instance FromHttpApiData CID where
+  parseUrlPiece = Right . CID
```

Comment:
Could do more verification?

Reply:
Whatever... at _worst_ we get an error from IPFS, which could happen even if it was properly formatted 🤷‍♀️